designer/public/js/Heart_health.js

```diff
@@ -0,0 +1,738 @@
+$(function() {
```

Comment:
all the code is written wrt NGSI-V1 which is not as per design . Need to update as per the design(protocol independent).